### PR TITLE
Feature: RFID Component

### PR DESCRIPTION
**Pull request by ppmathis**
This PR adds documentation for the component tracked as ppmathis/fhnw-crowpi#19 and implemented in Pi4J/pi4j-example-crowpi#3. Additionally I've fixed the components overview page stored in `crowpi.md`, it seems like the table ended up being duplicated there.

### CODE REVIEW

File: content/german/components/rfid.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+## Funktionsweise
+
+Bei RFID, abgekürzt für Radio Frequency Identification, handelt es sich um eine Technologie, welche es ermöglicht automatisch und kontaktlos
+mit einer Karte entsprechende Informationen auszutauschen. Dies wird zum Beispiel bei weit bekannten NFC Tags verwendet, jedoch auch für
+Anwendungen wie das kontaktlose Zahlen per Kreditkarte.
+
+Ein grosser Vorteil ist hierbei, dass die jeweiligen Karten/Tags, auch Transponder genannt, so klein wie ein Reiskorn sein können und keine
+eigene Stromversorgung benötigen. Im CrowPi ist die Komponente `MFRC522` verbaut, welche ein hochfrequentes elektronisches Wechselfeld bei
+13.56 MHz erzeugt, um damit einerseits die Karte mit Strom zu versorgen und andererseits mit dieser zu kommunizieren. Somit können
+beispielsweise die beiden beigelegten Tags des CrowPi ausgelesen und beschrieben werden mit beliebigen Daten.
```
Comment (tobiassiegrist) [inline content/german/components/rfid.md:15]:
```suggestion
beispielsweise die beiden beigelegten Tags des CrowPi ausgelesen und beschrieben werden. Dies funktioniert dabei mit beliebigen Daten.
```

File: content/german/components/rfid.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+Die Technologie dahinter ist sehr komplex und die entsprechenden Informationen dazu sind auf mehrere Standards aufgeteilt, hauptsächlich
+`ISO-14443` für den allgemein Aufbau und die Kommunikation sowie die entsprechenden Spezifikationen von NXP Semiconductors, dem Hersteller
```
Comment (tobiassiegrist) [inline content/german/components/rfid.md:18]: Do you have a link to ISO-14443?

**Review by tobiassiegrist — CHANGES_REQUESTED** [review]
Hi 
Just a few really small points. You did really your best on this RFID Component.
Thanks alot

File: content/german/components/rfid.md (hunk 2 above)
Comment (ppmathis) [reply content/german/components/rfid.md]: I've made some slight changes to the docs, what are your thoughts on the updated version?

**Review by tobiassiegrist — APPROVED** [review]
LGTM

**Pull request merged**